extensions/blocks/social-previews/editor.scss

```diff
@@ -12,3 +12,75 @@
 		fill: currentColor;
 	}
 }
+
+.jetpack-social-previews__upgrade-description {
+	margin-bottom: 1em;
+}
+
+.jetpack-social-previews__upgrade-heading {
+	font-size: 1.9em;
```

Comment:
Just for me learning up... How do you manage effortlessly using `em` when the design has `px` ❓